.github/build-android-project.yml

```diff
@@ -13,7 +13,7 @@
 
 name: "CI"
 
-on: [push, pull_request]
+on: [push, pull_request, workflow_dispatch, workflow_call]
```

Comment:
please revert changes to the workflow files. You can of course keep whatever personal you do....in your own branches.